### PR TITLE
Modify start script to use serve

### PR DESCRIPTION
**Pull request by AbegaM**
### Modifications

1. Installed the `serve` package
2. Modified the start script

### CODE REVIEW

**Comment by IanMayo** [comment]
so, we can remove the `http-server` dependency, right @AbegaM ?

File: public/index.html [hunk 1]
```diff
@@ -0,0 +1 @@
+View published output <a href="dita/out/index.html">here</a>.
```
Comment (IanMayo) [inline public/index.html:1]: I wasn't expecting this.  I guess it's not required now that we're hosting from `dita/out`.

Comment (AbegaM) [reply public/index.html:1]: yes, it is not required, do you want me to remove it?

Comment (IanMayo) [reply public/index.html:1]: yes please

**Review by IanMayo — APPROVED** [review]

**Pull request merged**